ledger-core/v2/application/functest/create_wallet_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+// +build functest
+
+package functest
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func getCount(args []string) (int, error) {
+	for _, s := range args {
+		if strings.Contains(s, "test.count") {
+			countInfo := strings.Split(s, "=")
+			return strconv.Atoi(countInfo[1])
+		}
+	}
+	return 1, nil
+}
+
+// Creates wallets and check Reference format in response body.
+func TestCreateWallet(t *testing.T) {
+	count, err := getCount(os.Args)
```

Comment:
Again, we shouldn't tie logic of tests to arguments of `go test`. Why do we call create API multiple times in a single test? Reasons:

- to repeat a fast operation a few times
- to get more chances to hit pulse change in action
- to hit server side with concurrent requests
- for other reasons you wouldn't get from repeating test using `-count N` argument of `go test`

How do we pick number of repeats? Large enough to increase number of events described above. Small enough to make sure one run doesn't go for ages. Number is test dependent and may change over time.

Reply:
I can get a number of working nodes from "node.GetStatus" and multiply it on custom value.
```
status := getStatus(t)
count := 50 * status.WorkingListSize
```
So, the test will depend on network size.

Reply:
I'm ok with just constant at this point.

Reply:
done